packages/search-adapter/BREAKING-CHANGES.md

```diff
@@ -0,0 +1,55 @@
+# Search Adapter Changelog
+
+## 6.0.0
```

Comment:
There is something weird here, on master the search-adapter project has the changelog updated up to the 6.1.0 version.